### PR TITLE
Switch to auto-formatted code with YAPF

### PR DESCRIPTION
**Pull request by bkgoksel**
This is the result of running YAPF (https://github.com/google/yapf) with Facebook style on all of our codebase. If this looks good, we could make YAPF formatting a standard and enforce compliance in CI.

### CODE REVIEW

File: codalab/bundles/run_bundle.py [hunk 1]
```diff
@@ -23,36 +23,214 @@ class RunBundle(DerivedBundle):
...
-    METADATA_SPECS.append(MetadataSpec('time', float, 'Amount of time (seconds) used by this run (total).', generated=True, formatting='duration'))
-    METADATA_SPECS.append(MetadataSpec('time_user', float, 'Amount of time (seconds) by user.', generated=True, formatting='duration'))
-    METADATA_SPECS.append(MetadataSpec('time_system', float, 'Amount of time (seconds) by the system.', generated=True, formatting='duration'))
-    METADATA_SPECS.append(MetadataSpec('memory', float, 'Amount of memory (bytes) used by this run.', generated=True, formatting='size'))
-    METADATA_SPECS.append(MetadataSpec('memory_max', float, 'Maximum amount of memory (bytes) used by this run at any time during execution.', generated=True, formatting='size'))
-
-    METADATA_SPECS.append(MetadataSpec('started', int, 'Time when this bundle started executing.', generated=True, formatting='date'))
-    METADATA_SPECS.append(MetadataSpec('last_updated', int, 'Time when information about this bundle was last updated.', generated=True, formatting='date'))
-    METADATA_SPECS.append(MetadataSpec('run_status', basestring, 'Execution status of the bundle.', generated=True))
+    METADATA_SPECS.append(
```
Comment (percyliang) [inline codalab/bundles/run_bundle.py:26]: Expanding these make it very hard to read and get a general sense of the args.

File: codalab/client/json_api_client.py [hunk 2]
```diff
@@ -491,21 +511,30 @@ def delete(self, resource_type, resource_ids, params=None):
...
                 query_params=self._pack_params(params),
-                data=data))
+                data=data
+            )
+        )
 
     @wrap_exception('Unable to create {1}/{2}/relationships/{3}')
-    def create_relationship(self, resource_type, resource_id, relationship_key,
-                            relationship, params=None):
+    def create_relationship(
```
Comment (percyliang) [inline codalab/client/json_api_client.py:530]: This feels unnecessary to me.

File: codalab/bin/cl.py [hunk 3]
```diff
@@ -59,22 +62,33 @@ def run_server_with_watch():
...
+        ),
         Commands.Argument(
-            '-p', '--processes',
+            '-p',
+            '--processes',
             help='Number of processes to use. A production deployment should '
-                 'use more than 1 process to make the best use of multiple '
-                 'CPUs.',
-            type=int, default=1),
+            'use more than 1 process to make the best use of multiple '
```
Comment (percyliang) [inline codalab/bin/cl.py:73]: No indent here feels wrong...

**Review by percyliang — COMMENTED** [review]
Overall, I think makes some good changes...but I think some places it causes too much wrapping.

File: codalab/bundles/run_bundle.py (hunk 1 above)
Comment (bkgoksel) [reply codalab/bundles/run_bundle.py:26]: For this kind of place we can manually override auto formatting.

**Comment by bkgoksel** [comment]
Closing as YAPF doesn't seem like a good fit.

**Pull request closed without merging**